web/app/js/components/StatusTable.jsx

```diff
@@ -42,7 +42,7 @@ const columns = {
       dataIndex: "name",
       key: "name",
       render: (name, record) => {
-        if (shouldLink && _.some(record.statuses, ["value", "good"])) {
+        if (shouldLink && _.some(record.statuses, o => o.value === 'good' || o.value === 'neutral')) {
```

Comment:
in `ServiceMesh.jsx` we have `return { name: p.name, value: p.added ? "good" : "neutral" };` which means pods can only have values good or neutral. so this `_.some` doesn't seem necessary any longer